relations/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Kisio Digital and/or its affiliates.
+//
+// This program is free software: you can redistribute it and/or
+// modify it under the terms of the GNU General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+
+//! Modeling the relations between objects.
+
+/// The error type used by the crate.
+pub type Error = failure::Error;
+
+/// The corresponding result type used by the crate.
+pub type Result<T> = std::result::Result<T, Error>;
+
+pub mod relations;
```

Comment:
Because of the line below, I think you don't have to make this one `pub`lic.
```suggestion
mod relations;
```